content/en/blog/_posts/2025-12-17-kubernetes-v1-35-release/index.md

```diff
@@ -0,0 +1,450 @@
+---  
+layout: blog
+title: "Kubernetes v1.35: TBD"
+date: 2025-12-17T10:30:00-08:00
+draft: true
+evergreen: true
+slug: kubernetes-v1-35-release
+author: >
+  [Kubernetes v1.35 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.35/release-team.md)
+---
+
+**Editors**: Aakanksha Bhende, Arujjwal Negi, Chad M. Crowell, Graziano Casto, Swathi Rao
+
+Similar to previous releases, the release of Kubernetes v1.35 introduces new stable, beta, and alpha features. The consistent delivery of high-quality releases underscores the strength of our development cycle and the vibrant support from our community.
+
+This release consists of 59 enhancements. Of those enhancements, 16 have graduated to Stable, 19 have entered Beta, and 22 have entered Alpha.
+
+There are also some [deprecations and removals](#deprecations-and-removals) in this release; make sure to read about those.
+
+## Release theme and logo
+
+{{< figure src="TBD" alt="Kubernetes v1.35 logo: TBD" class="release-logo" >}}
+
+TBD
+
+## Spotlight on key updates
+
+Kubernetes v1.35 is packed with new features and improvements. Here are a few select updates the Release Team would like to highlight!
+
+### Stable: In-place update of Pod resources
+
+Kubernetes is graduating in-place updates for Pod resources to General Availability (GA).
+This feature allows users to adjust cpu and memory resources without restarting Pods or Containers. Previously, such modifications required recreating Pods, which could disrupt workloads, particularly for stateful or batch applications. Previous Kubernetes releases already allowed you to change infrastructure resources settings (requests and limits) for existing Pods. This allows for smoother vertical scaling, improves efficiency, and can also simplify development.
+
+This work was done as part of [KEP #1287](https://kep.k8s.io/1287) led by SIG Node.
+
+### Beta: Pod certificates for workload identity and security
```

Comment:
Hello @kubernetes/sig-auth-leads @ahmedtd! Our release comms team put this section together for https://kep.k8s.io/4317 for the v1.35 announcement blog which we're sharing with the media for an early peak, and of course, we release next week.
How does this look?

Appreciate any contribution, advice or feedback 😄